modules/data/generic_pdf.components.json

```diff
@@ -37,5 +37,17 @@
         "selectorData": "download",
         "strategy": "id",
         "groups": []
+    },
+
+    "first-name-field": {
+        "selectorData": "pdfjs_internal_id_638R",
```

Comment:
I don't think I trust the PDFjs internal ids, I would change to `input[name='First Name Given Name']` with strategy `css`